app/models/incoming_message.rb

```diff
@@ -55,7 +55,8 @@ class IncomingMessage < ActiveRecord::Base
   # never really has many info_request_events, but could in theory
   has_many :info_request_events, :dependent => :destroy
 
-  belongs_to :raw_email, :dependent => :destroy
```

Comment:
Looks like this got fixed in 4.2. Search for `12380` in https://github.com/rails/rails/blob/4-2-stable/activerecord/CHANGELOG.md. Doesn't exist in https://github.com/rails/rails/blob/4-1-stable/activerecord/CHANGELOG.md